src/frolladaptive.c

```diff
@@ -26,13 +26,13 @@ void fadaptiverollmean(unsigned int algo, double *x, uint64_t nx, ans_t *ans, in
  */
 void fadaptiverollmeanFast(double *x, uint64_t nx, ans_t *ans, int *k, double fill, bool narm, int hasna, bool verbose) {
   if (verbose)
-    snprintf(end(ans->message[0]), 500, "fadaptiverollmeanFast: running for input length %"PRIu64", hasna %d, narm %d\n", (uint64_t)nx, hasna, (int) narm);
+    snprintf(end(ans->message[0]), 500, _("%s: running for input length %"PRIu64", hasna %d, narm %d\n"), "fadaptiverollmeanFast", (uint64_t)nx, hasna, (int) narm);
   bool truehasna = hasna>0;                                     // flag to re-run if NAs detected
```

Comment:
Ok understood re this change.  Will be interesting if this passes Rtools ATC re PR #4073 and the discussion here: https://github.com/Rdatatable/data.table/pull/4073#discussion_r355211599

Reply:
I ran it through win-builder ATC, and yes it passes with no warnings.  So this means it was something about the `__fun__` and not the `%s` before `PRIu64`.  I'll add a link to the previous discussion pointing here too.

Reply:
awesome, glad to hear